omaha/hammer.bat

```diff
@@ -5,7 +5,7 @@
 set OMAHA_PSEXEC_DIR=%ProgramFiles(x86)%\pstools
 
 :: Set VS environment variables.
-call "%ProgramFiles(x86)%\Microsoft Visual Studio\2017\Community\VC\Auxiliary\Build\vcvarsall.bat" amd64_x86 10.0.18362.0
+call "%ProgramFiles(x86)%\Microsoft Visual Studio\2017\Community\VC\Auxiliary\Build\vcvarsall.bat" amd64_x86 10.0.18362.0 || (SET ERRORLINE=8&& goto :error)
```

Comment:
nit: How about give one space like `SET ERRORLINE=8 && goto :error` for more readable?
Please ignore this if this is bat file's convention.

Reply:
Thank you for the review. With the space, it becomes part of the variable value. (`"8 "` instead of `"8"`). I didn't find a way to avoid this (quotation marks don't help, for example). So will leave it as-is.

Reply:
Oh, I didn't know that. Thanks for kind explanation! :)

Reply:
Sure! Thank you for the review :)